rustls/src/lib.rs

```diff
@@ -425,14 +420,14 @@ pub mod client {
     };
     pub use handy::ClientSessionMemoryCache;
 
-    #[cfg(feature = "dangerous_configuration")]
-    pub use crate::verify::{HandshakeSignatureValid, ServerCertVerified, ServerCertVerifier};
-    #[cfg(feature = "dangerous_configuration")]
-    pub use crate::webpki::{
-        verify_server_cert_signed_by_trust_anchor, verify_server_name, WebPkiServerVerifier,
-    };
-    #[cfg(feature = "dangerous_configuration")]
-    pub use client_conn::danger::DangerousClientConfig;
+    /// Dangerous configuration that should be audited and used with extreme care.
+    pub mod danger {
+        pub use super::client_conn::danger::DangerousClientConfig;
+        pub use crate::verify::{HandshakeSignatureValid, ServerCertVerified, ServerCertVerifier};
+        pub use crate::webpki::{
+            verify_server_cert_signed_by_trust_anchor, verify_server_name, WebPkiServerVerifier,
+        };
```

Comment:
I think I agree with @djc that these types (and `server::danger`) are not inherently dangerous.

Reply:
I've removed these types from the `danger `mod.